src/app/analyze/stack/stack-details/stack-details.component.html

```diff
@@ -3,166 +3,203 @@
-               src="icons/266-question.svg">
-             </img> -->
-          </a>
-        </h4>
-        <div id="licenses">
-          <p> Licenses Found: {{totalLicenses}}</p>
-          <p> Distinct Licenses: {{distinctLicenses.length }} </p>
-          <p> Licenses: {{distinctLicenses}} </p>
+        <div class="card-pf card-pf-accented card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#licenses-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Licenses</a>
+          </h2>
+          <div class="card-pf-body">
+
+            <div id='licenses'>
+              <p> Licenses Found: {{totalLicenses}}</p>
+              <p> Distinct Licenses: {{distinctLicenses.length }} </p>
+              <p> Licenses: {{distinctLicenses}} </p>
+            </div>
+            <p class="card-pf-view-notifications">
+            </p>
+          </div>
         </div>
       </div>
       <div class="col-sm-6 col-lg-6 render-stack-tiles ">
-        <h4>
-          <span class="nav-text">Security</span>
-          <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#security-stack">
-            <!-- <img title='CVEs found in this stack. Click for more info.'
-              src="icons/266-question.svg">
-            </img> -->
-          </a>
-        </h4>
-        <div id="security">
-          <p>Total CVEs: {{totalSecurityIssues}}</p>
-          <p>CVSS: {{cvss}}</p>
+        <div class="card-pf card-pf-accented card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#security-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Security</a>
+          </h2>
+          <div class="card-pf-body">
+            <div id='security '>
+              <p>Total CVEs: {{totalSecurityIssues}}</p>
+              <p>CVSS: {{cvss}}</p>
+            </div>
+            <p class="card-pf-view-notifications">
+            </p>
+          </div>
         </div>
       </div>
     </div>
 
-
-
     <div class="row">
       <div class="col-sm-6 col-lg-6 render-stack-tiles ">
-        <h4><span class="nav-text">Metadata</span>
-          <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#metadata-stack">
-            <!-- <img title="Info from metadata files. Click for more info."
-              src="icons/266-question.svg"></img> -->
-          </a>
-        </h4>
-        <div id="metadata">
+        <div class="card-pf card-pf-accented card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#metadata-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Metadata</a>
```

Comment:
Why do you have reference to https://gitlab.cee.redhat.com here. This is not a public URL. It does not work on this publicly running application.